frontend/src/scenes/billing/BillingProduct.tsx

```diff
@@ -200,13 +203,21 @@ const BillingProduct = ({ product }: { product: BillingProductV2Type }): JSX.Ele
                                 <div className="space-y-2">
                                     <LemonLabel
                                         info={
-                                            'This is roughly calculated based on your current bill and the remaining time left in this billing period.'
+                                            predictUsageWillBeLimited
+                                                ? `This is roughly calculated based on your current bill, the remaining time left in this billing period, and your billing limit. Without a limit you could expect to pay $${product.projected_amount_usd} for this billing period.`
+                                                : `This is roughly calculated based on your current bill and the remaining time left in this billing period.`
                                         }
                                     >
                                         Predicted bill
                                     </LemonLabel>
                                     <div className="font-bold text-muted text-2xl">
-                                        ${product.projected_amount_usd || '0.00'}
+                                        $
+                                        {
+                                            // if the billing limit for the product is less than the projected amount, show the limit instead
+                                            predictUsageWillBeLimited
+                                                ? billing?.custom_limits_usd?.[product.type]
+                                                : product.projected_amount_usd || '0.00'
```

Comment:
do we also need to check against how much they've already used, in the case that they set their custom limit to be less than what they've used for the month?

Reply:
Ah, you're right. 😑 will update.